server.js

```diff
@@ -213,32 +229,27 @@ app.get('/about', function(req,res) {
     return res.render('about', { user: req.session.user });
 });
 
-/* Fetch and render the leaderboard table. Send HTML as response. */
+/* Fetch and render the leaderboard table.*/
 app.get('/leaderboard-table', function(req, res) {
     if (!req.session.user) {
         return res.redirect('/');
     }
-
-    var fullTable = true;
-    var shortTable = false;
-
-    if (req.query.fullTable === 'false') {
-        fullTable = false;
-    }
-
-    if (req.query.longTable === 'false') {
-        shortTable = true;
+    var smallBoard = false;
+    if (req.query.smallBoard === 'true'){
+        smallBoard = true;
     }
-
-    users.getLeaderboard(req.session.user._id, shortTable, function(leader) {
-        var html = leaderboardTablePug({
-            fullTable: fullTable,
-            shortTable: shortTable,
-            leaderboard: leader,
-            userid: req.session.user._id
+    users.getLeaderboard(req.session.user._id, smallBoard, function(leader, displayRank) {
```

Comment:
Check for err, also what does leader mean?

Reply:
Fixed, leader was the list of students I was getting from the db. I've given it a better name, it is now leaderboardList.